package.json

```diff
@@ -15,7 +15,8 @@
     "storybook:build": "build-storybook -s ./docs/assets -o ./docs/storybook",
     "test": "jest",
     "test:watch": "jest --watch",
-    "test:coverage": "jest --coverage"
+    "test:coverage": "jest --coverage",
+    "chromatic": "chromatic --project-token CHROMATIC_PROJECT_TOKEN --exit-zero-on-changes"
```

Comment:
is this env var evaluated? looks more like a static project token to me

Reply:
I guess the arguments given in https://github.com/webex/components/pull/217/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R54 overwrite the ones here anyway, so maybe it would be better to just remove them from here:
```json
"chromatic": "chromatic"
```
unless I'm missing something:)

Reply:
This token is taken from environment variables and it helps to trigger the script manually, I guess we don't need it anymore. I added it just for testing as from here I couldn't access the token saved in circleCI